[2025-03-04T03:16:18.240090+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://127.0.0.1:8000/"" at RouterListener.php line 149 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://127.0.0.1:8000/\" at F:\\wamp64\\www\\chess\\ChessSfVue\\back\\vendor\\symfony\\http-kernel\\EventListener\\RouterListener.php:149)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\NoConfigurationException(code: 0):  at F:\\wamp64\\www\\chess\\ChessSfVue\\back\\vendor\\symfony\\routing\\Matcher\\Dumper\\CompiledUrlMatcherTrait.php:181)"} []
[2025-03-04T03:17:43.800009+00:00] request.CRITICAL: Uncaught PHP Exception InvalidArgumentException: "The controller for URI "/api/authentication_token" is not callable: Controller "lexik_jwt_authentication.controller.token" does neither exist as service nor as class." at ControllerResolver.php line 97 {"exception":"[object] (InvalidArgumentException(code: 0): The controller for URI \"/api/authentication_token\" is not callable: Controller \"lexik_jwt_authentication.controller.token\" does neither exist as service nor as class. at F:\\wamp64\\www\\chess\\ChessSfVue\\back\\vendor\\symfony\\http-kernel\\Controller\\ControllerResolver.php:97)\n[previous exception] [object] (InvalidArgumentException(code: 0): Controller \"lexik_jwt_authentication.controller.token\" does neither exist as service nor as class. at F:\\wamp64\\www\\chess\\ChessSfVue\\back\\vendor\\symfony\\http-kernel\\Controller\\ContainerControllerResolver.php:52)\n[previous exception] [object] (Error(code: 0): Class \"lexik_jwt_authentication.controller.token\" not found at F:\\wamp64\\www\\chess\\ChessSfVue\\back\\vendor\\symfony\\http-kernel\\Controller\\ControllerResolver.php:152)"} []
[2025-03-04T03:19:51.788429+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: ""App\Entity\User" object not found by "Symfony\Bridge\Doctrine\ArgumentResolver\EntityValueResolver"." at EntityValueResolver.php line 76 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): \"App\\Entity\\User\" object not found by \"Symfony\\Bridge\\Doctrine\\ArgumentResolver\\EntityValueResolver\". at F:\\wamp64\\www\\chess\\ChessSfVue\\back\\vendor\\symfony\\doctrine-bridge\\ArgumentResolver\\EntityValueResolver.php:76)"} []
